awscli/argparser.py

```diff
@@ -25,7 +33,7 @@
 )
 USAGE = (
     "aws [options] <command> <subcommand> [<subcommand> ...] [parameters]\n"
-    "%s" % HELP_BLURB
+    "%s\n%s" % (HELP_BLURB, AWS_CLI_V2_MESSAGE)
```

Comment:
What do you think about moving this to the beginning of the error output? That way its the first thing they see, and then its not taking up space between their error message and the usage guidance.